src/app/pages/sharing/smb/smb-form/smb-form.component.ts

```diff
@@ -180,6 +180,13 @@ export class SMBFormComponent implements FormConfiguration, OnDestroy {
           tooltip: helptext_sharing_smb.tooltip_timemachine,
           isHidden: true,
         },
+        {
+          type: 'checkbox',
+          name: 'afp',
+          placeholder: helptext_sharing_smb.placeholder_afp,
+          tooltip: helptext_sharing_smb.tooltip_afp,
+          isHidden: true,
```

Comment:
If it is hidden by default. What is the condition to show?

Reply:
We probably should.

Reply:
Line 333. UpdateForm method toggles hidden based on Basic vs Advanced mode
This is why the entire 'other' fieldset has every field's isHidden set the same way.